app/core/src/main/java/com/fsck/k9/preferences/GeneralSettingsDescriptions.java

```diff
@@ -283,6 +283,9 @@
         s.put("showRecentChanges", Settings.versions(
                 new V(73, new BooleanSetting(true))
         ));
+        s.put("showStarredCount", Settings.versions(
+                new V(73, new BooleanSetting(false))
```

Comment:
Regarding the version, see:

https://github.com/k9mail/k-9/blob/f6e9948c2ad3b4056592fb77015b0da0f3907cfb/app/core/src/main/java/com/fsck/k9/preferences/GeneralSettingsDescriptions.java#L47-L50

Reply:
Ok. Done. (Bumped to version 75)